rest-api/etl/source_to_concept_map.csv

```diff
@@ -24,7 +26,7 @@ BlackSpecific_Liberian|1585618|ppi-race|Liberian|38003598|Race|1970-1-1|2099-12-
 BlackSpecific_Nigerian|1585624|ppi-race|Nigerian|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_Somali|1585619|ppi-race|Somali|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_SouthAfrican|1585620|ppi-race|South African|38003598|Race|1970-1-1|2099-12-31|\N|1
-Hispanic_HispanicSpecific|1586156|ppi-race|Hispanic, Latino, or Spanish (For example: Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.) *|0|\N|\N|\N|\N|2
```

Comment:
So it seems that these are more specific codes for hispanic race, but we are missing two top-level race codes in this file: WhatRaceEthnicity_Hispanic and WhatRaceEthnicity_RaceEthnicityNoneOfThese.

It seems like the former should map to hispanic ethnicity, and should be included in this file. I'm less sure about how to handle the second.

Reply:
Also: I think(?) we may need to keep this row as it was before, and add additional row(s) with source_vocabulary_id set to "ppi-ethnicity" or something.

We want to be able to set race_source_concept_id to the concept IDs for these rows, while setting ethnicity_concept_id to the right ethnicity... those seem like they have to be separate rows?

Reply:
I see. So two rows with different vocabulary ids.